packages/web/src/components/gcds-footer/gcds-footer.tsx

```diff
@@ -66,29 +61,51 @@ export class GcdsFooter {
     return !!this.el.querySelector('[slot="list"]');
   }
 
+  private get hasContextual() {
+    return !!this.el.querySelector('[slot="contextual"]');
+  }
+
   render() {
-    const { lang, display, topHref, hasList, renderSignature } = this;
+    const { lang, display, hasList, hasContextual, renderSignature } = this;
     const govNav = I18N[lang].gov.menu;
+    const themeNav = I18N[lang].themes.menu;
     const siteNav = I18N[lang].site.menu;
 
     return (
       <Host role="contentinfo">
         <div class="top__container">
           <slot name="top"></slot>
```

Comment:
Should we remove this slot as it is most likely no longer needed?

Reply:
Agreed, will remove top slot.